From c1fc9246a1121b59c64b13f7a4d6f15cfc0a26ea Mon Sep 17 00:00:00 2001
From: Tom Finegan <tomfinegan@chromium.org>
Date: Fri, 14 Dec 2012 14:36:24 -0800
Subject: [PATCH] FFmpeg: Add support for VP9 in WebM.

BUG=166094
TEST=libavformat codec context for VP9 in matroska has codec_id
AV_CODEC_ID_VP9

Signed-off-by: Dale Curtis <dalecurtis@chromium.org>
---
 libavcodec/avcodec.h   | 1 +
 libavformat/matroska.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/libavcodec/avcodec.h b/libavcodec/avcodec.h
index b6a4b0e..aa06348 100644
--- a/libavcodec/avcodec.h
+++ b/libavcodec/avcodec.h
@@ -267,6 +267,7 @@ enum AVCodecID {
     AV_CODEC_ID_MTS2,
     AV_CODEC_ID_CLLC,
     AV_CODEC_ID_MSS2,
+    AV_CODEC_ID_VP9,
     AV_CODEC_ID_BRENDER_PIX= MKBETAG('B','P','I','X'),
     AV_CODEC_ID_Y41P       = MKBETAG('Y','4','1','P'),
     AV_CODEC_ID_ESCAPE130  = MKBETAG('E','1','3','0'),
diff --git a/libavformat/matroska.c b/libavformat/matroska.c
index 64d0a45..930c609 100644
--- a/libavformat/matroska.c
+++ b/libavformat/matroska.c
@@ -82,6 +82,7 @@ const CodecTags ff_mkv_codec_tags[]={
     {"V_THEORA"         , AV_CODEC_ID_THEORA},
     {"V_UNCOMPRESSED"   , AV_CODEC_ID_RAWVIDEO},
     {"V_VP8"            , AV_CODEC_ID_VP8},
+    {"V_VP9"            , AV_CODEC_ID_VP9},
 
     {""                 , AV_CODEC_ID_NONE}
 };
-- 
1.7.11.4

